README.md

```diff
@@ -155,6 +155,22 @@ For instance, you can ignore all the npm modules:
 ignoreFilenames: ['node_modules'],
 ```
 
+### `additionalLibraries`
+
+This option gives the possibility to remove other `propTypes` in addition to the canonical `prop-types`.
+
+For instance, by default
+```js
+import PropTypes from 'prop-types'
+import ImmutablePropTypes from 'react-immutable-proptypes'
+```
+will result in `propTypes` not to be removed; with:
```

Comment:
I'm not sure we are on the same page. Here the output of your test with the following option change.
**option**
```diff
{
- "additionalLibraries": ["react-immutable-proptypes"],
  "removeImport": true
}

```

**output**
```diff
import React, { Component } from 'react';

-import ImmutablePropTypes from 'react-immutable-proptypes';

export default class Greeting extends Component {
 constructor(props, context) {
   super(props, context);
```